data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,436 @@
+			choice
+				`	(Approach the pair.)`
+				`	(Keep walking.)`
+					goto walking
+			
+			`	As you approach the human, a relatively young-looking man, glances up at you. "Oh hello, just arrived did you? You didn't happen to see an Argosy out there on your way in?"`
+			`	"Oh shush," says the Hai, "Your uncle will be here when said he will, no need to bother random folks just going about their business."`
+			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition didn't you. Can we help you at all Captain?"`
+			choice
+				`	"Is this a common problem for Allhome?"`
+				`	"I heard you say this happened on Solace?"`
+					goto solace
+					
+			`	"Oh imperishables, no, not at all. It's completely unprecedented for something like that to happen, and in broad daylight too from the looks of the footage," she says.`
+			`	"It's like I was saying before, Auntie, ever since the retraction and that advisory, I swear no one except pirates is trying to get through the wormhole anymore."`
+			`	He looks at you seriously, "It's getting so bad that it's starting to feel like it did during the war, when all the militia were away and before the Free Worlds got a handle on the pirates. It seemed like every new ship that arrived was there to try and rob us, and it's the same now. We came here to be safe, but now it feels like all the government has managed to do is make sure that no one friendly is willing to risk the journey..."`
+				goto resolve
+			
+			label solace
+			`	The guy sits up a little straighter. "When the war started and most of the militia went away to the front, before the Free Worlds got a handle on the pirates, Solace essentially came under siege. Other than the regular merchants that we were familiar with, it seemed like every new ship that arrived was there to try and rob us. Either raiding the regulars, or raiding the planet.`
+			`	"And now with all this," he says, gesturing to the screen. "It feels much the same now, and the Hai seem no better prepared for it than we were. Ever since the retraction and that advisory, it feels like pirates are the only ones showing up through the wormhole anymore. We came here to be safe, but now it feels like all the government has managed to do is make sure that no one friendly is willing to risk the journey..."`
+			
+			label resolve
+			`	His lip quivers and she takes his hand firmly in her own. "You are safe with your Uncle and I," she says, forcefully emphasizing the word 'are', "it will take far more than some pressure from a few pirates to threaten the peace here."`
+			`	He nods and stands up, heading towards the restroom. "Excuse me Auntie, I need a minute."`
+			
+			choice
+				`	"I'll leave you in peace."`
+					goto peace
+				`	"Will he be ok?"`
+					goto "him ok"
+				`	"Are you ok?"`
+					goto "she ok"
+				`	"You're his Aunt?"`
```

Comment:
I don't think 'uncle' and 'aunt' should be capitalised like titles such as 'Captain'. Especially if 'mother' and 'father' aren't capitalised later on.
```suggestion
			`	His lip quivers and she takes his hand firmly in her own. "You are safe with your uncle and I," she says, forcefully emphasizing the word 'are', "it will take far more than some pressure from a few pirates to threaten the peace here."`
			`	He nods and stands up, heading towards the restroom. "Excuse me auntie, I need a minute."`
			
			choice
				`	"I'll leave you in peace."`
					goto peace
				`	"Will he be ok?"`
					goto "him ok"
				`	"Are you ok?"`
					goto "she ok"
				`	"You're his aunt?"`
```

Reply:
One should "Capitalize family relationship titles when they are used with names or in place of names."

So you're right for all of these except "Auntie" which is always used to address her directly and thus substitutes for a name.

Reply:
```suggestion
			`	His lip quivers and she takes his hand firmly in her own. "You are safe with your uncle and I," she says, forcefully emphasizing the word 'are', "it will take far more than some pressure from a few pirates to threaten the peace here."`
			`	He nods and stands up, heading towards the restroom. "Excuse me Auntie, I need a minute."`
			
			choice
				`	"I'll leave you in peace."`
					goto peace
				`	"Will he be ok?"`
					goto "him ok"
				`	"Are you ok?"`
					goto "she ok"
				`	"You're his aunt?"`
```